<ui version="4.0" >
 <author></author>
 <comment>*********************************************************************
  **
  ** Copyright (C) 1992-$THISYEAR$ Trolltech AS. All rights reserved.
  **
  ** This file is part of the $MODULE$ of the Qt Toolkit.
  **
  ** $LICENSE$
  **
  ** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
  ** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
  **
  *********************************************************************</comment>
 <exportmacro></exportmacro>
 <class>PaletteEditorBase</class>
 <widget class="QWidget" name="PaletteEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Palette</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="paletteLabel" >
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>1</y>
           <width>78</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text" >
          <string>Select Palette:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="paletteCombo" >
         <property name="geometry" >
          <rect>
           <x>85</x>
           <y>8</y>
           <width>115</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text" >
           <string>Active Palette</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Inactive Palette</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Disabled Palette</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="geometry" >
          <rect>
           <x>206</x>
           <y>1</y>
           <width>91</width>
           <height>40</height>
          </rect>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="paletteTab" >
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>49</y>
         <width>298</width>
         <height>299</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>4</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="tab" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>314</width>
          <height>290</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Simple</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>1</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="simpleGroup" >
           <property name="geometry" >
            <rect>
             <x>1</x>
             <y>1</y>
             <width>312</width>
             <height>74</height>
            </rect>
           </property>
           <property name="title" >
            <string>Build Palette</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>11</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="EffectLabel" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>23</y>
                <width>140</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text" >
               <string>3-D Effects:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="BackgroundLabel" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>45</y>
                <width>140</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text" >
               <string>Background:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="StyleButton" name="effectButton" >
              <property name="geometry" >
               <rect>
                <x>159</x>
                <y>23</y>
                <width>800</width>
                <height>480</height>
               </rect>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="StyleButton" name="backgroundButton" >
              <property name="geometry" >
               <rect>
                <x>159</x>
                <y>45</y>
                <width>0</width>
                <height>0</height>
               </rect>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="geometry" >
            <rect>
             <x>1</x>
             <y>81</y>
             <width>312</width>
             <height>208</height>
            </rect>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>314</width>
          <height>290</height>
         </rect>
        </property>
        <attribute name="title" >
         <string>Advanced</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>1</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="autoGroup" >
           <property name="geometry" >
            <rect>
             <x>1</x>
             <y>1</y>
             <width>312</width>
             <height>111</height>
            </rect>
           </property>
           <property name="title" >
            <string>Auto</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>11</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="inactiveCheckBox" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>23</y>
                <width>286</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text" >
               <string>Build the inactive palette from active palette.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="disabledCheckBox" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>50</y>
                <width>286</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text" >
               <string>Build the disabled palette from active palette.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="buttonCheckBox" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>77</y>
                <width>286</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text" >
               <string>Build 3-D shadow effects from button color.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="rolesGroup" >
           <property name="geometry" >
            <rect>
             <x>1</x>
             <y>118</y>
             <width>312</width>
             <height>85</height>
            </rect>
           </property>
           <property name="title" >
            <string>Color Roles</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>11</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QComboBox" name="rolesCombo" >
              <property name="geometry" >
               <rect>
                <x>13</x>
                <y>23</y>
                <width>286</width>
                <height>25</height>
               </rect>
              </property>
              <item>
               <property name="text" >
                <string>Central Role -> Background</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Central Role -> Foreground</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Central Role -> Base</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Central Role -> Text</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Central Role -> Button</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Central Role -> ButtonText</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>3-D Effect -> Light</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>3-D Effect -> Midlight</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>3-D Effect -> Mid</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>3-D Effect -> Dark</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>3-D Effect -> Shadow</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Selected Item -> Highlight</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Selected Item -> HighlightText</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Special Role -> BrightText</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Special Role -> Link</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Special Role -> LinkVisited</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>1</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="colorLabel" >
                <property name="geometry" >
                 <rect>
                  <x>1</x>
                  <y>1</y>
                  <width>67</width>
                  <height>16</height>
                 </rect>
                </property>
                <property name="text" >
                 <string>Color:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="StyleButton" name="advColorButton" >
                <property name="geometry" >
                 <rect>
                  <x>74</x>
                  <y>1</y>
                  <width>0</width>
                  <height>0</height>
                 </rect>
                </property>
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>4</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="pixmapLabel" >
                <property name="geometry" >
                 <rect>
                  <x>146</x>
                  <y>1</y>
                  <width>67</width>
                  <height>16</height>
                 </rect>
                </property>
                <property name="text" >
                 <string>Pixmap:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="StyleButton" name="advPixmapButton" >
                <property name="geometry" >
                 <rect>
                  <x>219</x>
                  <y>1</y>
                  <width>0</width>
                  <height>0</height>
                 </rect>
                </property>
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>4</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="geometry" >
            <rect>
             <x>1</x>
             <y>209</y>
             <width>312</width>
             <height>80</height>
            </rect>
           </property>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>7</y>
           <width>77</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text" >
          <string>OK</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="geometry" >
          <rect>
           <x>84</x>
           <y>1</y>
           <width>132</width>
           <height>40</height>
          </rect>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="geometry" >
          <rect>
           <x>222</x>
           <y>8</y>
           <width>75</width>
           <height>26</height>
          </rect>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="previewGroup" >
     <property name="geometry" >
      <rect>
       <x>317</x>
       <y>11</y>
       <width>195</width>
       <height>397</height>
      </rect>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="PreviewWidget" name="previewWindow" >
        <property name="geometry" >
         <rect>
          <x>4</x>
          <y>14</y>
          <width>800</width>
          <height>480</height>
         </rect>
        </property>
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends></extends>
   <header>previewwidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>StyleButton</class>
   <extends>QWidget</extends>
   <header>stylebutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
